docs/source/tutorials/object_detection.ipynb

```diff
@@ -498,6 +509,115 @@
     "visualize(image_path)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "46d6282a-4601-4cc3-b8a8-187ea6d5f8bc",
+   "metadata": {},
+   "source": [
+    "## Exploratory data analysys\n",
+    "\n",
+    "This section focuses on techniques to uncover annotation irregularities through exploratory data analysis. Specifically, it aims to highlight anomalies in object sizes, detect images with unusual object counts, and examine the distribution of class labels."
+   ]
+  },
+  {
```

Comment:
add text cell before this code cell, which says:

"Let's first consider the number of objects per image, and inspect the images with the largest values:"